ci-operator/config/openshift/installer/openshift-installer-master__terraform.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  base:
+    name: "4.15"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.15
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: images/terraform-providers/Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.20-openshift-4.15:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.15
+  to: installer-terraform-providers
```

Comment:
> /hold [openshift/installer#7716](https://github.com/openshift/installer/pull/7716) is renaming the dir where the Dockerfile is.

Yes, it looks like it takes some effort to get an image promoted, so I think we should rename the image to `installer-infrastructure-providers` or something more generic, so that we will not need to update the image if we want to use the same pattern for CAPI provders .